dogs.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+   <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title>Adopt A Dog</title>
+      <link rel="stylesheet" href="stylesheet/style.css">
+   </head>
+   <body>
+      <header>
+         <div class="header-container">
+            <div class="logo-bar">
+               <img src="images/adoptadog-logo.svg" alt="Adopt A Dog Logo">
+            </div>
+            <div class="navigation-bar">
+               <nav>
+                  <ul>
+                     <li><a href="index.html">Home / </a></li>
```

Comment:
see if you can use css pseudo classes to handle these `/` elements. You'll need to look up the `after` element and use the  `content` propery